### DISCUSSION TITLE
Device rebooting

### DISCUSSION THREAD

**Discussion by XownedXIV** [afpineda/OpenSourceSimWheelESP32#6] [Q&A] [answered]
Flashed a ESP32E Devkitc V4 with the setup 2 sketch and this is the error that keeps popping up in serial monitor. Any ideas as to what can cause this? 
16:57:41.792 -> ELF file SHA256: a38963478ee8a382
16:57:41.792 -> 
16:57:41.872 -> Rebooting...
16:57:41.872 -> ets Jul 29 2019 12:21:46
16:57:41.872 -> 
16:57:41.872 -> rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
16:57:41.916 -> configsip: 0, SPIWP:0xee
16:57:41.916 -> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
16:57:41.916 -> mode:DIO, clock div:1
16:57:41.916 -> load:0x3fff0030,len:1344
16:57:41.916 -> load:0x40078000,len:13964
16:57:41.916 -> load:0x40080400,len:3600
16:57:41.916 -> entry 0x400805f0
16:57:42.135 -> 
16:57:42.135 -> abort() was called at PC 0x400d1d4f on core 1
16:57:42.135 -> 
16:57:42.135 -> 
16:57:42.135 -> Backtrace: 0x4008394d:0x3ffc8570 0x40092b25:0x3ffc8590 0x40098039:0x3ffc85b0 0x400d1d4f:0x3ffc8630 0x400d23be:0x3ffc8670 0x400d240d:0x3ffc8690 0x400e42ce:0x3ffc86f0

**Comment by afpineda** [comment, marked as answer]
In order to troubleshoot, first, set "Core Debug Level" to "Error" in Arduino IDE (board settings).
Then, upload your sketch again. 
This time, you will get more information from the serial monitor. 
Then, post your findings here.

**Comment by afpineda** [comment]
Just to be sure:
I uploaded "Setup2.ino" to an ESP32-DevKit-C board with firmware version 4.1.3.
No error messages at boot and advertising as a BLE device.
Seems to work as expected.
